source/blog/2017-01-09-ember-2-11-released.md

```diff
@@ -138,17 +138,148 @@ and [addons here](https://github.com/ember-cli/ember-addon-output/compare/v2.10.
+build when you publish your application; otherwise you will also include the
+assets from `ember-welcome-page` into your application.
+
+#### Other Notable Changes
+
+We have a litany of other smaller improvements in this release:
+
+- We [watch the `vendor` folder by default](https://github.com/ember-cli/ember-cli/pull/6436),
+making rebuilds work for changes in that directory. This may have performance
+consequences, please monitor the resource consumption in your applications to
+ensure that we have not regressed.
+- Stefan Penner and David Hamilton made it so we do a better job at
+[cleanup upon exit of Ember CLI](https://github.com/ember-cli/ember-cli/pull/6423)
+preventing pollution of the `tmp` folder inside of your applications.
+- Robert Jackson dramatically [reduced the number of merge steps](https://github.com/ember-cli/ember-cli/pull/6453)
+inside of the build, speeding up the build process.
 
 For more details on the changes in Ember CLI 2.11 and detailed upgrade
 instructions, please review the [Ember CLI 2.11.0 release page](https://github.com/ember-cli/ember-cli/releases/tag/v2.11.0).
 
-### Upcoming changes in Ember CLI 2.12
-
-TBK
+### Upcoming Changes in Ember CLI 2.12
+
+We adopted the standard six week release cycle for Ember CLI shortly following
+EmberConf 2016. This constant cadence allows us to get the improvements we've
+collectively been working on into our applications. Ember CLI 2.12 will be the
+most work we've ever had in a six week release and we're ecstatic to get all
+297 commits (not including merges or upstream changes!) into your hands.
+
+#### Babel
+
+We've made changes in the way you configure Babel in your applications and also
+now require that addons which need Babel transpilation supply their own
+`ember-cli-babel` dependency. The root application is _no longer_ wholly
+responsible for the transpilation of all dependent addons. There is a
+deprecation message which should guide you through the steps to make the
+necessary changes in your application.
+
+We no longer overload the `babel` key inside of `ember-cli-build.js` to
+conditionally use certain arguments for `babel` and others for
+`ember-cli-babel`. This is categorically better, and we've provided deprecation
+messages to help guide you to the new correct usage pattern.
+
+Rather than trying to walk through how to make these changes, please test this
+out during the beta period and let us know if the messages are able to guide you
+to the newly correct setup.
+
+#### ESLint All The Things!
+
+Tobias Bieniek has been on a mission; we've now completed the move to ESLint
+as the newly recommended linting tool for Ember applications. After upgrading to
+Ember CLI 2.12 and running `ember init` you will be presented with an option to
+remove the existing `ember-cli-jshint` and adopt `ember-cli-eslint`. The process
+should be relatively seamless, please test it out and let us know how it works
+in your applications during the beta period.
```

Comment:
- the option that you get is whether it should automatically remove the remaining `.jshintrc` files, but I think `ember-cli-jshint` will be removed unconditionally
- "The process should be relatively seamless" ... unless you made any changes to the defaults and have to port your settings from JSHint to ESLint. Calling it seamless might not be the right term for some people 😉